### ISSUE TITLE
provide granular ACLs for zookeeper

### ISSUE THREAD

**Issue by siggy** [BuoyantIO/linkerd#279]
our zk code writes everything as `Data.ACL.AnyoneAllUnsafe`. consider providing more granular control, both for the dcos bootstrapping configuration as well as the dtab storage code.

motivated by: https://github.com/BuoyantIO/linkerd/pull/271/files#r59498272

**Comment by adleong** [comment]
Does this specifically pertain to the DCOS bootstrap?  The regular ZkDtabStore supports configurable ACLs.

**Comment by siggy** [comment]
Yeah, `ZkDtabStore` got configurability after this ticket was written:
https://github.com/BuoyantIO/linkerd/commit/cb502a0f645135e265d670b94ba60dd680e2c1df

I've updated the description accordingly.

I think it's still worth considering for `DcosBootstrap`. I think we could make it a DCOS package config option on install. Would require changes in this repo, as well as in https://github.com/mesosphere/universe/blob/version-2.x/repo/packages/N/namerd/0/config.json.
